MAINTAINERS.md

```diff
@@ -46,7 +46,6 @@ for general contribution guidelines.
 - [vishalpowar](https://github.com/vishalpowar), Google LLC
 - [wenbozhu](https://github.com/wenbozhu), Google LLC
 - [yashykt](https://github.com/yashykt), Google LLC
-- [yijiem](https://github.com/yijiem), Google LLC
```

Comment:
I think there's a process that needs to be followed for this step. Please do it separately.

Reply:
Reverting those changes. What's the process? I haven't seen one in my ~5 years here.